.github/workflows/release-base.yml

```diff
@@ -33,3 +33,6 @@ jobs:
       - name: Build and push docker image
         working-directory: ./docker
         run: make build-release-base
+      - name: Build and push server base image
+        working-directory: ./docker
+        run: make release-base-server
```

Comment:
Add condition check for `build-image` function to avoid duplicate docker building?
https://github.com/star-whale/starwhale/blob/79b106c3ab10b92c767d3017e3e99dde1466b5af/docker/Makefile#L93-L94

Reply:
done in Makefile